### ISSUE TITLE
How to map a specific version for "semanticEquivalent"? 

### ISSUE THREAD

**Issue by UlrikeS91** [openMetadataInitiative/openMINDS#34]
I'm working on the mapping of schema.org vocab to openMINDS vocab. There are many different versions for the model, I'm comparing  their latest version (v28.0) at the moment. Therefore, we should have the version in the mapping somehow. 

If they were to change any properties or types in the future, this specific version mapping would remain accurate. 

What do you think?

**Comment by lzehl** [comment]
There should be one type with version information so the mapping is always type to schema.org and the version should be irrelevant for this mapping.

**Comment by lzehl** [comment]
related to #33

**Comment by UlrikeS91** [comment]
I'm leaning towards removing the 'semanticEquivalent' completely.

I've started the work of mapping this for schema.org and, as we already discussed, came across a few issues. Shortly summarized, a single property as we envisioned will be highly inaccurate and raises the question of how useful will this be. The other way around, we would have to increase the complexity and add additional properties to make this mapping more accurate and useful. Then the question is, how much work do we want to put into this? And at which point do we use too much time? Time that could be used to make a full mapping to design a converter. 
This mapping would be a great starting point for anybody interested in creating a converter, but I do not want to provide partly inaccurate content (e.g., mapping schema.org's 'amount' to our 'amount' which is semantically the same, but schema.org defines it very differently). Unless we find a good solution for this problem that is not too time consuming, I do not think we should do this kind of mapping at all.

**Comment by lzehl** [comment]
@UlrikeS91 let's bring this up again in the next dev meeting. I also tend towards removing the semanticEquivalent from the vocab and only keeping track within the vocab of our version mapping (which is difficult enough).

**Comment by Raphael-Gazzotti** [comment]
I completely agree. Including this in our vocab files seems unnecessary since it won't be utilized by external sources. However, maintaining a mapping within the OpenMINDS schemas is highly valuable and relevant.

**Comment by lzehl** [comment]
@openMetadataInitiative/openminds-developers common decision is to kill the semantic equivalent from vocab and move this effort to the mappings for the converters.

For mapping in converters look again at other efforts:
RDF mapping language
FAIR impact project (part of EOSC)

**Comment by lzehl** [comment]
TODO: PR for removing "semanticEquivalent" from vocab

**Comment by Raphael-Gazzotti** [comment]
Can we close this one?
